pullrequest.rst

```diff
@@ -119,10 +119,27 @@ homepage.
 Submitting
 ----------
 
-Once you are satisfied with your work you will want to commit your
-changes to your branch. In general you can run ``git commit -a`` and
-that will commit everything. You can always run ``git status`` to see
-what changes are outstanding.
+Once you are satisfied with your work, you will need to commit your
+changes to your branch. Assume you want to add a file which you have changed,
+you can add it to the list of changes to be committed (the "staging area")::
+
+   git add <file>
```

Comment:
I think this level of detail is not needed. GitHub has pretty good introductory level resources ([example](https://services.github.com/on-demand/downloads/github-git-cheat-sheet.pdf)) about using Git and I prefer to refer the readers to them.